opds.py

```diff
@@ -200,7 +200,15 @@ def categories(cls, work):
     @classmethod
     def authors(cls, work, license_pool, edition, identifier):
         """Create one or more <author> tags for the given work."""
-        return [AtomFeed.author(AtomFeed.name(edition.author or ""))]
+        authors = list()
+        for author in edition.author_contributors:
+            authors.append(AtomFeed.author(AtomFeed.name(
+                author.display_name or author.sort_name
+            )))
```

Comment:
We sometimes have a problem where the same contributor (or possibly a different contributor with the same name) is listed as a contributor twice. It won't fix the problem, but putting a dupe check in this code is a really easy way to stop the problem from affecting patrons. Keep track of the names we've used, and skip an author if we've used their name before.